src/app/pages/quickMaps/pages/baselineDetails/foodItems/foodItems.component.scss

```diff
@@ -1,42 +1 @@
-@import 'src/assets/scss/_colour';
```

Comment:
did you intent to remove all scss except a single commented out line from fooditems?

Reply:
Yeah.  I include a second default scss file in all those cards now so this one is only for component specific scss, of which there is none.

I like to leave the colors there to prompt its use! Although I often comment it out.